docgen/assets/js/examples/tourism/index.js

```diff
@@ -0,0 +1,291 @@
+import {InstantSearch, Hits, SearchBox, Pagination, Range, RefinementList, NumericRefinementList} from 'react-instantsearch';
```

Comment:
Did you not use react-hot-loader on purpose or you maybe not have seen how we use it elsewhere?